src/pages/discovery/loadingAnimation/loadingAnimation.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./loadingAnimation.scss";
+import loadingAnimation from "../../../assets/loadingAnimationAssets/loadingAnimation.svg";
+
+/**
+ * Renders a centered loading animation when active. To use, initialize with 'loading' prop, and set to true for display,
+ * and false for none.
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const LoadingAnimation = (props) => {
+    return(
+        <img
+            src={loadingAnimation}
+            id="discovery-loader"
+            class={(props.dataLoaded) ? "discovery-not-loading" : "discovery-loading" }
```

Comment:
will need to be updated to work with the new boolean defaults and new variable name. Will also need to generalize the class names since this may be used site wide.